### ISSUE TITLE
expose template function

### ISSUE THREAD

**Issue by jfirebaugh** [mapbox/mapbox.js#385]
If you're doing a custom grid interaction, you probably still want to use mustache templates defined via TileMill or elsewhere:

```
var map = L.mapbox.map('map', 'examples.map-8ced9urs', {gridControl: false});

var popup = new L.Popup({ autoPan: false });
map.gridLayer.on('click', function(e) {
    if (e.data) {
        popup.setContent(L.mapbox.template(map.getTileJSON().template, e.data))
            .setLatLng(e.latLng)
            .openOn(map);
    } else {
        map.closePopup();
    }
});
```

This method would also accept a `sanitizer` option.

**Comment by jfirebaugh** [comment]
Also expose `L.mapbox.sanitize`.

**Comment by jfirebaugh** [comment]
Closed in favor of #437.
